### PR TITLE
More Views

### PR DESCRIPTION
**Pull request by gycn**

### CODE REVIEW

File: app/src/js/drawable/3d/label3d_handler.ts [hunk 1]
```diff
@@ -206,21 +227,52 @@ export class Label3DHandler {
...
-            Session.dispatch(selectLabel(
-              Session.label3dList.selectedLabelIds, -1, -1
-            ))
-          } else {
-            Session.dispatch(selectLabel(
-              Session.label3dList.selectedLabelIds,
-              this._selectedItemIndex,
-              this._plane.labelId
-            ))
+        {
```
Comment (fyu) [inline app/src/js/drawable/3d/label3d_handler.ts]: Since we have to put paratheses here, a separate function can make the code easier to write and read.

Comment (gycn) [reply app/src/js/drawable/3d/label3d_handler.ts]: I made a separate function

**Comment by smajida** [comment]
is the fusion working?@fyu

**Comment by fyu** [comment]
@smajida not yet.

**Pull request merged**